otter/models/cass.py

```diff
@@ -641,7 +649,12 @@ def wrapper(*args):
             return d
         return wrapper
 
-    def view_manifest(self, with_policies=True, with_webhooks=False):
+    def _group_status(self, status, deleting):
```

Comment:
While collecting metrics, I found many groups that had incorrect launch config or the tenant itself was fake. So, I changed the status of these groups to `DISABLED` and [filtered](https://github.com/rackerlabs/otter/blob/master/otter/metrics.py#L266) these out from when collecting metrics. I was not sure what statues we all decide on at that time. I think changing these groups to ERROR is fine.